pennylane_lightning/core/src/simulators/lightning_kokkos/bindings/LKokkosBindings.hpp

```diff
@@ -238,6 +238,12 @@ auto getBackendInfo() -> py::dict {
  * @param m Pybind11 module.
  */
 void registerBackendSpecificInfo(py::module_ &m) {
+    m.def("kokkos_initialize", []() { Kokkos::initialize(); });
+    m.def("kokkos_initialize",
+          [](const InitializationSettings &args) { Kokkos::initialize(args); });
+    m.def("kokkos_finalize", []() { Kokkos::finalize(); });
+    m.def("kokkos_is_initialized", []() { return Kokkos::is_initialized(); });
+    m.def("kokkos_is_finalized", []() { return Kokkos::is_finalized(); });
```

Comment:
What is the relation between this and the GPU testing?

Reply:
This is useful in accessing Kokkos functionality without creating a device (for example if device creation crashes Python).

Reply:
Adding GPU testing required some debugging, which was made easier allowing Kokkos init/finalization independently from device creation. I believe this will remain handy, also to set global Kokkos options (GPU device ID, number of OpenMP threads) via `InitializationSettings`.

Reply:
Makes sense.